sdk/storage/storage-file-datalake/test/perfstress/track-2/read.spec.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { streamToBuffer3, StorageDFSTest } from "./storageTest.spec";
+import { DataLakeFileClient } from "../../../src";
+interface StorageDFSReadTestOptions {
+  size: number;
+}
+
+export class StorageDFSReadTest extends StorageDFSTest<StorageDFSReadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageDFSReadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+  static fileName = `newfile${new Date().getTime()}`;
+  fileClient: DataLakeFileClient;
+
+  constructor() {
+    super();
+    this.fileClient = this.directoryClient.getFileClient(StorageDFSReadTest.fileName);
+  }
+
+  public async globalSetup() {
+    await super.globalSetup();
+    // await this.fileClient.create();
+    await this.fileClient.upload(Buffer.alloc(this.parsedOptions.size.value!));
+  }
+
+  async runAsync(): Promise<void> {
+    const ReadResponse = await this.fileClient.read();
+    await streamToBuffer3(ReadResponse.readableStreamBody!);
```

Comment:
It looks like `streamToBuffer3()` creates a new `Buffer` everytime, and we don't want to include this in our perf testing, since what if the customer consumes the stream directly?  Can we instead read all the bytes from the `readableStreamBody`, but just throw them away instead of allocating a new buffer?  In .NET, this would be:

```C#
readableStream.CopyTo(Stream.Null);
```

Reply:
Migrated to the `drainStream` instead https://github.com/Azure/azure-sdk-for-js/pull/13014